components/organisms/SignUpForm/SignUpForm.js

```diff
@@ -1,16 +1,38 @@
-import React from 'react';
+import React, { useState } from 'react';
+import styled from 'styled-components';
 
 import SignUpFormContent from './SignUpFormContent';
 import useSignUp from './useSignUp';
 
+export const StyledFormWrapper = styled.div`
+  text-align: center;
+`;
+
+export const StyledToggleForm = styled.div`
+  display: inline-block;
+  cursor: pointer;
+  color: #00f;
+  margin-top: 1rem;
+`;
+
 const SignUpForm = ({ onSuccess }) => {
   // todo remove callbacks from mutation hooks
   const {
     signUp,
     mutationResult: { error, loading },
   } = useSignUp({ onSuccess });
 
-  return <SignUpFormContent error={error} loading={loading} onSubmit={signUp} />;
+  const [activeForm, toggleSignForm] = useState('signUp');
+  const handleActiveFormChoose = () => {
+    activeForm === 'signUp' ? toggleSignForm('signIn') : toggleSignForm('signUp');
+  };
+  const toggleButtonText = activeForm === 'signUp' ? 'Already signed up' : 'Sign up';
+  return (
+    <StyledFormWrapper>
+      <SignUpFormContent error={error} loading={loading} onSubmit={signUp} activeForm={activeForm} />
+      <StyledToggleForm onClick={() => handleActiveFormChoose()}>{toggleButtonText}</StyledToggleForm>
```

Comment:
You can just pass a handler without wrapping in another arrow function `onClick={handleActiveFormChoose}`